adafruit_connection_manager.py

```diff
@@ -58,6 +58,10 @@ def __init__(self, socket: CircuitPythonSocketType, tls_mode: int) -> None:
         self.recv = socket.recv
         self.close = socket.close
         self.recv_into = socket.recv_into
+        if hasattr(socket, "_interface"):
+            self._interface = socket._interface
+        if hasattr(socket, "_socket_pool"):
+            self._socket_pool = socket._socket_pool
```

Comment:
Also would it be smaller to use `getattr()` here?

Reply:
Are you suggesting:
```
        self._interface = getattr(socket, "_interface", None)
        self._socket_pool = getattr(socket, "_socket_pool", None)
```
It wouldn't hurt. If using the `fona` or an old version of `esp32spi`: since it doesn't have these, they would just get set to `None`. They aren't used previously so it wouldn't break anything

Reply:
yes, just slightly smaller amount of code.

Reply:
fixed here: b14ed99